microsoft-edge/webview2/concepts/working-with-local-content.md

```diff
@@ -786,6 +794,12 @@ HRESULT AppWindow::WebResourceRequestedEventHandler(
 ---
 
 
+<!-- ====================================================================== -->
+## Custom scheme registration
```

Comment:
Will the added content be an h2 section?  
Will the h2 section be at end of article?
How does custom scheme registration fit in w/ rest of article?
@aluhrs13

Reply:
Probably should be an H2, @yildirimcagri-msft is probably the right person to work with on the details of this, I'm not super familiar with using it myself.

Reply:
new section is h4 within existing h2 section.